docs/ref/templates/builtins.txt

```diff
@@ -957,6 +957,83 @@ output (as a string) inside a variable. This is useful if you want to use
     {% now "Y" as current_year %}
     {% blocktranslate %}Copyright {{ current_year }}{% endblocktranslate %}
 
+.. templatetag:: partial
+
+``partial``
+-----------
+
+.. versionadded:: 6.0
+
+Renders a partial that was defined with :ttag:`partialdef`.
+
+Usage:
+
+.. code-block:: html+django
+
+    {% partial partial_name %}
+
+The ``partial_name`` argument is the string identifier of the partial to render.
+
+Example:
+
+.. code-block:: html+django
+
+    {% partialdef button %}
+        <button class="btn">{{ button_text }}</button>
+    {% endpartialdef %}
+
+    {% partial button %}
+    {% partial button %}
```

Comment:
Why two? shouldn't `button` be a string to not make the docs confusing? How about `button_text` , where is that defined?

Side note: I know this is a ref doc so we shouldn't go into too much detail about examples, but we either provide a simple but functional example or we link to a how-to or a more hands on doc (or both!).

Reply:
<img width="786" height="288" alt="image" src="https://github.com/user-attachments/assets/24e52196-3bec-4ef3-8251-c0a5785a73e1" />
Something like this.